src/Fhir.Anonymizer.Shared.Core/Processors/IAnonymizerProcessor.cs

```diff
@@ -5,6 +5,6 @@ namespace Fhir.Anonymizer.Core.Processors
 {
     public interface IAnonymizerProcessor
     {
-        public ProcessResult Process(ElementNode node);
+        public ProcessResult Process(ElementNode node, ProcessSetting setting = null);
```

Comment:
We can spilt it to 2 parameters: settings (dictionary<string, object>) and context (for visited nodes..., FHIR path...)

Reply:
Thanks, fixed